tools/operator/testdata/kafka-leveldb-notls-network-spec.yml

```diff
@@ -152,4 +152,4 @@ k8s:
 #!  consortium: SampleConsortium
 #!  organizations:
 #!    - org3
-#!    - org4
```

Comment:
Check other files. The last few reviews have some using a Newline on the last line, and some do not. Does it matter? If so, then take a look at all our yml files so they are all standard and correct.

Reply:
I will make them consistent